packages/react-scripts/scripts/test.js

```diff
@@ -31,15 +31,36 @@ if (process.env.SKIP_PREFLIGHT_CHECK !== 'true') {
 // @remove-on-eject-end
 
 const jest = require('jest');
+const execSync = require('child_process').execSync;
```

Comment:
Should this use `react-dev-utils/crossSpawn`?

Reply:
I dunno. I copy pasted from `init.js`. That's what we use there. Should I worry?

Reply:
We also use it in CRA itself for detecting yarn. I think it's fine.